### PR TITLE
new remediation - CCE-27567-7, disable ctl+alt+del

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

**Comment by shawndwells** [comment]
Bump on this. I believe it's ready to be merged.

File: RHEL/6/input/fixes/bash/disable_ctrlaltdel_reboot.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+sed -i 's,^exec.*,exec /usr/bin/logger -p authpriv.notice -t init "Ctrl-Alt-Del was pressed and ignored",'  FILE
```
Comment (redhatrises) [inline RHEL/6/input/fixes/bash/disable_ctrlaltdel_reboot.sh]: @shawndwells `FILE` should be `/etc/init/control-alt-delete.override` and should probably have a `cp -p /etc/init/control-alt-delete.conf /etc/init/control-alt-delete.override` before the `sed` command.

**Comment by shawndwells** [comment]
thanks @redhatrises, updated. can't believe I missed that!

File: RHEL/6/input/fixes/bash/disable_ctrlaltdel_reboot.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+if [ -f /etc/init/control-alt-delete.conf ]; then
```
Comment (redhatrises) [inline RHEL/6/input/fixes/bash/disable_ctrlaltdel_reboot.sh]: @shawndwells shouldn't it be `if [ ! -f /etc/init/control-alt-delete.override ]`?

**Comment by redhatrises** [comment]
@shawndwells testing this I believe that a `$` is missing after the `*` which is preventing the script from working exactly.

**Comment by redhatrises** [comment]
Fixed the regex and merged. Thanks for the PR!

**Pull request merged**